lean4-infoview/src/infoview/rpcSessions.tsx

```diff
@@ -180,7 +180,8 @@ export class RpcSessions implements Disposable {
                 if (!this.#connecting.has(uri)) {
                     // force a reconnect.
                     if (this.#connected.has(uri)) {
-                       this.#connected.delete(uri);
+                        this.#connected.get(uri)?.dispose()
```

Comment:
Thanks! This was definitely an oversight.

Reply:
yep, my fault I think... since I've been running with this fix I have not seen the pesky output errors...